.github/workflows/code-formating-check.yml

```diff
@@ -0,0 +1,29 @@
+name: "Check Formating"
+on: [push, pull_request]
+
+jobs:
+  check_code_formatting:
+    name: Check code formatting with Black
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
```

Comment:
does it need to run on multiple python versions?

Reply:
So far it is running only on version 3.8

Reply:
I don't think it has to run on multiple versions. This is just the version that executes black, not the python version the syntax is checked for.

Reply:
I thought so, so let's remove the build matrix...

Reply:
The filename is misspelled, it should be `code-formatting-check.yml`